contracts/dao/Implementation.sol

```diff
@@ -23,19 +23,33 @@ import "./Regulator.sol";
 import "./Bonding.sol";
 import "./Govern.sol";
 import "../Constants.sol";
+import "../external/AggregatorV3Interface.sol";
 
 contract Implementation is State, Bonding, Market, Regulator, Govern {
     using SafeMath for uint256;
 
     event Advance(uint256 indexed epoch, uint256 block, uint256 timestamp);
     event Incentivization(address indexed account, uint256 amount);
 
-    function initialize() initializer public {
-        // committer reward:
-        mintToAccount(msg.sender, 150e18); // 150 DSD to committer
-        // contributor  rewards:
-        mintToAccount(0xF414CFf71eCC35320Df0BB577E3Bc9B69c9E1f07, 1000e18); // 1000 DSD to devnull
+    function initialize() public initializer {
+        oracle().capture(); // capture for pool price on sushi pool
```

Comment:
@devnulldsd, this should only be called by the DAO. So it will work only after the vote when governance calls `updateTo()`.

Reply:
initializer checks for this